src/org/mozilla/javascript/EqualObjectGraphs.java

```diff
@@ -37,6 +39,19 @@
 final class EqualObjectGraphs {
     private static final ThreadLocal<EqualObjectGraphs> instance = new ThreadLocal<>();
 
+    private static final Set<Class<?>> valueClasses =
+            Collections.unmodifiableSet(
+                    new HashSet<>(
+                            Arrays.asList(
```

Comment:
This is the closest we can get to a `Set.of` in a pre-9 Java; currently Rhino still compiles for 8.